### PR TITLE
console: add support forums UI

### PR DESCRIPTION
**Pull request by surendran82**
### Description
console support forum ui update

### Changelog

- [ ] `CHANGELOG.md` is updated with user-facing content relevant to this PR. If no changelog is required, then add the `no-changelog-required` label.

### Affected components

- [x] Console

### Related Issues

close https://github.com/hasura/graphql-engine-internal/issues/392

### CODE REVIEW

File: console/src/components/Main/Help.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+import close from './images/x-circle.svg';
+import ToolTip from '../Common/Tooltip/Tooltip';
+import Toggle from '../Common/Toggle/Toggle';
+import {
+  closeIntercom,
+  startIntercom,
+  persistChatState,
+  getPersistedChatState,
+} from './utils';
+import { useToggle } from '../../hooks/useToggle';
```
Comment (soorajshankar) [inline console/src/components/Main/Help.tsx:16]: :tada:

File: console/src/components/Main/utils.ts [hunk 2]
```diff
@@ -1,3 +1,46 @@
...
+  script.type = 'text/javascript';
+  script.onload = bootIntercom;
+  script.src = INTERCOM_URL;
+  head.appendChild(script);
+};
+
+const chatState = 'console:chatEnabled';
+
+export const persistChatState = (isEnabled: boolean) => {
+  window.localStorage.setItem(chatState, String(isEnabled));
```
Comment (soorajshankar) [inline console/src/components/Main/utils.ts:29]: we have a utility for this `getLocalStorageItem`, we can re-use if you want, do as you wish :see_no_evil:

Comment (beerose) [reply console/src/components/Main/utils.ts:29]: I know, but... LS storage utils are all over the place and it needs some bigger cleanup. Will do it one day in another PR.

File: console/src/components/Services/Data/constants.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
 export const Operators = [
+  { name: 'is null', value: '$is_null', graphqlOp: '_is_null' },
```
Comment (beerose) [inline console/src/components/Services/Data/constants.js:2]:
```suggestion
```

**Review by rikinsk — APPROVED** [review]

**Review by beerose — APPROVED** [review]

**Pull request merged**